### PR TITLE
feat(ejs): package level custom .editorconfig

### PR DESCRIPTION
**Pull request by peterblazejewicz**
The DT project's Prettier uses global .editorconfig, this change allows to
use package-level editor config settings, here TABS (vs. SPACES):
https://git.io/Jv8Ar

/cc @ExE-Boss

Thanks!

- [x] Use a meaningful title for the pull request. Include the name of the package modified.

### CODE REVIEW

File: types/ejs/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[*]
+indent_style = tab
+indent_size = 4
```
Comment (ExE-Boss) [inline types/ejs/.editorconfig]: This line is unnecessary with tabs:
```suggestion
```

**Review by ExE-Boss — APPROVED** [review]
While I do approve of this, the @DefinitelyTyped team has rejected my PRs for containing this file in the past.

**Comment by PranavSenthilnathan** [comment]
Let's just have one prettier config for the whole repo. We prefer having a single consistent style across Definitely Typed as much as possible.

**Comment by peterblazejewicz** [comment]
> Let's just have one prettier config for the whole repo. We prefer having a single consistent style across Definitely Typed as much as possible.

The point of EditorConfig is to allow composition (one root, other are composed). Here we have just tried to avoid mistake I've done updating the types/ejs (converted tabs to spaces). This local editor config effects only the package (types/ejs) format.
Just discussing this, no problem with revisiting that in the future.
Thanks!

**Pull request closed without merging**